mdx-parse-server.js

```diff
@@ -1,12 +1,6 @@
 const mdx = require('@mdx-js/mdx');
 const http = require('http')
 
-async function parseMDX(file) {
```

Comment:
this was not used?

Reply:
No. It is old code copied from another file.